subcommand/acl-controller/command.go

```diff
@@ -88,26 +88,23 @@ func (c *Command) run() error {
 			CAPem: []byte(caCert),
 		}
 	}
-	if c.flagPartitionsEnabled {
-		cfg.Partition = c.flagPartition
-	}
```

Comment:
I think this is fine (I saw the previous discussion). Are we confident we've passed the partition everywhere it's needed in requests @cthain?

Reply:
@pglass Yes I'm quite confident that we've got all the boxes ticked w.r.t. adding the partition (and namespace) to the queries were necessary. The unit tests are fairly comprehensive and they fail in the enterprise case when the queries aren't qualified with partition and ns.